### ISSUE TITLE
Turn on Android 11+ bubble style for Facebook Messenger

### ISSUE THREAD

**Issue by kongacute** [revanced/revanced-patches#2255]
### Application

Facebook Messenger

### Issue

Some device will only have Android bubble and some have old style Messenger Chat Head.

### Patch

Force Messenger use Android Bubble instead of old Chat Head.

### Motivation

Some people like to use Android bubble. There are already have patch for that but doesn't allow to export apk file after patch.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
What is messenger chat head

**Comment by HypeMaxers** [comment]
![Screenshot_20230525_215947_One UI Home](https://github.com/revanced/revanced-patches/assets/41132838/05403438-741a-43ed-bc06-ec60c6a64d13)
@oSumAtrIX this. It's messenger's old way of having the new android chat bubbles style.

**Comment by oSumAtrIX** [comment]
Am I understanding correctly, you want to enable a feature that is present in newer Android versions in old versions of Android?

**Comment by kongacute** [comment]
@oSumAtrIX no. I mean, force it to use a new bubble on Android 11+. Because it randomly decides whether the device can use it or not. You can see MRV Patch. It currently provides a patch for that.

**Comment by kongacute** [comment]
> You can do it via LSPosed: https://github.com/NeonOrbit/ChatHeadEnabler

I don't root my device. So, non root solution is my needs. There are MRVPatcher can do the job, but I need to patch twice per update to get both ads block and bubble.
